external/storm-hdfs-blobstore/src/test/java/org/apache/storm/hdfs/blobstore/BlobStoreTest.java

```diff
@@ -204,15 +204,6 @@ public void testMultipleHdfs()
         }
     }
 
-    @Test
-    public void testHdfsWithAuth()
```

Comment:
This test was migrated to the parameterized test version below (`testWithAuthentication`)